### PR TITLE
Bump symfony to 3.4.8 and other pending minor bumps

### PR DESCRIPTION
**Pull request by phil-davis**
These are minor bumps that are waiting. ``composer update`` on ``,master``
As usual, I happen to notice these when rebasing the guzzle change.

symfony 3.4.8
zendframework/zend-filter 2.8.0
zendframework/zend-stdlib 3.1.1

No big deal, but saves ``dependabot`` a bit of bother.

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@phil-davis thanks for your effort on this - the dependency bot should do this - it'll run once a week and update our dependencies.

Don't waste your valuable time with this ;-)

Or did the bot again not grab this ......

**Comment by phil-davis** [comment]
The symfony release was early April, I think. And I did not see a dependabot PR for it yet.
I notice it when the Guzzle PR gets ``composer.lock`` conflicts. It only takes 5 minutes to knock out a separate PR like this to get waiting bumps done.

File: composer.lock [hunk 1]
```diff
@@ -1688,20 +1689,20 @@
             "keywords": [
                 "exception"
             ],
-            "time": "2015-02-10T20:07:52+00:00"
+            "time": "2015-02-10 20:07:52"
         },
         {
             "name": "phpseclib/phpseclib",
-            "version": "2.0.10",
+            "version": "2.0.11",
```
Comment (DeepDiver1975) [inline composer.lock]: #31152 #31153

Comment (phil-davis) [reply composer.lock]: Yes, this PR includes whatever little things are waiting for a general ``composer update`` to be done, without touching ``composer.json``
So ``dependabot`` will close its stuff if it sees this merged.

**Review by DeepDiver1975 — APPROVED** [review]

**Pull request merged**